lib/src/html/templates.dart

```diff
@@ -60,9 +60,9 @@ const _requiredTemplates = <String>[
   'typedef.html',
 ];
 
-const String _headerPlaceholder = '<!-- header placeholder -->';
-const String _footerPlaceholder = '<!-- footer placeholder -->';
-const String _footerTextPlaceholder = '<!-- footer-text placeholder -->';
+const String _headerPlaceholder = '{{! header placeholder }}';
```

Comment:
Might be worth a TODO for replacePlaceholder to port these to mustache rather than using our own template substitutions.  A typo in the template would be less easy to miss that way.